senza/cli.py

```diff
@@ -250,6 +252,56 @@ def __init__(self, **kwargs):
             setattr(self, key, val)
 
 
+class AccountArguments:
+    def __init__(self, region, **kwargs):
+        setattr(self, '__Region', region)
+        for key, val in kwargs.items():
+            setattr(self, key, val)
+
+    def __getAccountID(self):
+        attr = getattr(self, '__AccountID', None)
+        if attr is None:
+            accountid = get_account_id()
+            setattr(self, '__AccountID', accountid)
+            return accountid
+        return attr
+
+    def __getAccountAlias(self):
+        attr = getattr(self, '__AccountAlias', None)
+        if attr is None:
+            accountalias = get_account_alias()
+            setattr(self, '__AccountAlias', accountalias)
+            return accountalias
+        return attr
+
+    def __getRegion(self):
+        return getattr(self, '__Region', None)
+
+    def __getDomain(self):
+        attr = getattr(self, '__Domain', None)
+        if attr is None:
+            conn = boto3.client('route53')
+            domain = conn.list_hosted_zones()['HostedZones'][0]['Name']
```

Comment:
I guess this will only work for "plain vanilla" STUPS accounts with only one hosted zone..